shared/domains/chelonia/GIMessage.js

```diff
@@ -407,7 +406,7 @@ export class GIMessage {
     return `${desc}|${this.hash()} of ${this.contractID()}>`
   }
 
-  isFirstMessage (): boolean { return !this.head().previousHEAD }
+  isFirstMessage (): boolean { return !this.head().contractID }
```

Comment:
Why was this changed?

Reply:
Because I removed a bunch of the repetitive code in `chelonia.js` that was looking up the previous HEAD, which is now done by `publishEvent`. Since the previous HEAD could now be null, the check as it was would fail. Note that either condition is correct (if the previous HEAD were set).